ksp_plugin/plugin.hpp

```diff
@@ -221,17 +221,17 @@ class Plugin {
   virtual void AdvanceTime(Instant const& t, Angle const& planetarium_rotation);
 
   // Returns the degrees of freedom of the given part in |World|, assuming that
-  // the origin of |World| is fixed at |bubble_barycentre_|.  The part must be
-  // in a loaded vessel.
```

Comment:
It was, this invariant is not enforced, and has no fundamental reason to be. Eventually we will distinguish loaded and packed and we can think about that again.